### DISCUSSION TITLE
🇪🇸 List of resources in Spanish language

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/courses#32] [Ideas]
English is not the only language in the world. Many people do not speak English. Many creators do not make content in English. Therefore, we want to create separate lists of resources that can help people who want to build ML knowledge and do not speak that language. Spanish is the next language we want to support.

I am looking for people who would like to help me. I don't know Spanish 😞 I need your help to make it happen.

**Comment by datacubeR** [comment]
Hi @SkalskiP. 
So as far as I understand, you need not only translating the README but also blogs, videos, and things related in Spanish?

**Reply by SkalskiP** [reply]
Hi, @datacubeR 👋🏻! 

Thanks a lot for your interest in this project. Yes, that is exactly right. I want to create a separate list of resources with useful resources on ML/AI in each language. So people who do not speak English could also use this repository to build their knowledge.
